storage/src/mmr/journaled.rs

```diff
@@ -242,18 +248,18 @@ impl<B: Blob, E: RStorage<B> + Clock + Metrics, H: Hasher> Mmr<B, E, H> {
         }
```

Comment:
Should we do self.size() above?

Reply:
yes indeed; search 'n' replace is hard for me I guess.